week2/homework/exercises/1-blog-API/server.js

```diff
@@ -1,10 +1,78 @@
+const { Console } = require('console');
 const express = require('express')
 const app = express();
- 
+const fs = require("fs")
+const path = require("path")
 
 // YOUR CODE GOES IN HERE
-app.get('/', function (req, res) {
-  res.send('Hello World')
+app.use(express.json())
+
+app.get("/", (req, res) => {
+  res.send("Hello world")
 })
- 
-app.listen(3000)
\ No newline at end of file
+
+app.get('/blogs', function (req, res) {
+  fs.readdir(path.join(__dirname, "blogs"), "utf8", (error, content) => {
+    if (error) console.log(error);
+    res.send(content);
```

Comment:
Dont forget to send a response even if there is an error otherwise the client will wait forever!
```suggestion
    if (error) { 
       console.error(error);
       response.status(500).send('An unexpected error has occurred!');
    } else {
        res.send(content);
    }
```

Reply:
Thank you for the feedback, I will keep it in mind